slyd/app/components/annotations-plugin/component.js

```diff
@@ -584,6 +593,28 @@ export default Ember.Component.extend({
         }
     },
 
+    setIgnores: function() {
```

Comment:
You can make this into a shared variable stored in `pluginState` to match the behaviour of `maxVariant` and `maxSticky`